src/GraphQL/Types/ObjectGraphType.cs

```diff
@@ -7,6 +7,7 @@ namespace GraphQL.Types
 {
     public interface IObjectGraphType : IComplexGraphType, IImplementInterfaces
     {
+        Type SourceType { get; }
```

Comment:
Not every `IObjectGraphType` is going to have a `SourceType` that is helpful so I'm not sure that this is the direction that we should take.